snappi_ixnetwork/trafficitem.py

```diff
@@ -2157,4 +2157,277 @@ def rocev2_flow_results(self, request):
                         # TODO print a warning maybe ?
                         pass
         return list(flow_rows.values())
+    
+    def delete_configs(self, delete_flows_config):
+        """
+        Delete the flows with list of flow names
+        """
+        self._validate_delete_flows_config(delete_flows_config)
+
+        # Stop traffic before deletion of flows
+        start_states = [
+            "txStopWatchExpected",
+            "locked",
+            "started",
+            "startedWaitingForStats",
+            "startedWaitingForStreams",
+            "stoppedWaitingForStats",
+        ]
+        state = self._api._ixnetwork.Traffic.State
+        if state in start_states:
+            self._api._ixnetwork.Traffic.StopStatelessTrafficBlocking()
+
```

Comment:
According to our Athena-G discussion, We were decide to check that the traffic actually stopped.

Reply:
For implicit call we are checking whether flows are in started state or not, if it is we stopped all the traffic before append/delete flows otherwise it will throw error from ixnetwork.